pipenv/utils/resolver.py

```diff
@@ -5,7 +5,7 @@
 import sys
 import tempfile
 import warnings
-from functools import cached_property, lru_cache
+from functools import lru_cache
```

Comment:
Too many side-effects encountered in various issue reports relate to our current structure and usage of caching.  I am in favor of correcting core functionality, and revisiting small gaps in performance down the road in a more thoughtful way.